system-int-test/src/cmr/system_int_test/data2/umm_json.clj

```diff
@@ -152,6 +157,36 @@
              (set (map result-item-for-comparison
                        (get-in search-result [:results :items]))))))))
 
+(defn- service->umm-json-meta
+  "Returns the meta section of umm-json format."
+  [service]
+  (concept->umm-json-meta :service service))
+
+(defn- service->umm-json
+  "Returns the UMM JSON result of the given service."
+  [version service]
+  (if (:deleted service)
+    {:meta (service->umm-json-meta service)}
+    (let [;; use the original metadata for now, add version migration when service versioning is added
+           {:keys [metadata]} service]
+      {:meta (service->umm-json-meta service)
+       :umm (json/decode metadata true)})))
+
+(defn assert-service-umm-jsons-match
+  "Returns true if the UMM service umm-jsons match the umm-jsons returned from the search."
+  [version services search-result]
+  (if (and (some? (:status search-result)) (not= 200 (:status search-result)))
```

Comment:
Why have this if-else? Was it because too much was being printed to the REPL when the status code is not 200?

Reply:
This is copy-pasted from collection UMM JSON function. The if-else is to check for more things when the status is 200 or there is no status.